.github/renovate.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": [
+    "github>rancher/renovate-config#release"
+  ],
+  "baseBranches": [
+    "release/v0.3"
+  ],
+  "prHourlyLimit": 2,
+  "enabledManagers": [
+    "droneci",
+    "dockerfile",
+    "github-actions",
+    "helm-values",
+    "regex"
```

Comment:
It doesn't look like we are using regex https://docs.renovatebot.com/modules/manager/regex/

Reply:
The regex are defined at `https://github.com/rancher/renovate-config/blob/main/default.json`.